### PR TITLE
Create mamma-mia.md

### PR DESCRIPTION
**Pull request by arnoudplantinga**

### CODE REVIEW

File: rulebooks/mamma-mia.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
+---
+title: Mamma Mia!
+source: https://boardgamegeek.com/filepage/14388/mamamiahtml
```
Comment (camden) [inline rulebooks/mamma-mia.md]: Can you add the publisher? https://rulebook.io/how-to-help#publisher

File: rulebooks/mamma-mia.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+tags:
+    - card game
+information:
+    player-count: 2-5
+    play-time: 30 minutes
+---
+
+Contents
+========
```
Comment (camden) [inline rulebooks/mamma-mia.md]: Can you replace these lines (and other occurrences) with the `#` markdown notation?

File: rulebooks/mamma-mia.md (hunk 1 above)
Comment (arnoudplantinga) [reply rulebooks/mamma-mia.md]: Done!

File: rulebooks/mamma-mia.md (hunk 2 above)
Comment (arnoudplantinga) [reply rulebooks/mamma-mia.md]: Done!

**Pull request merged**